src/services/stringCompletions.ts

```diff
@@ -276,6 +288,8 @@ function stringLiteralCompletionDetails(name: string, location: Node, completion
         }
         case StringLiteralCompletionKind.Types:
             return find(completion.types, t => t.value === name) ? createCompletionDetails(name, ScriptElementKindModifier.none, ScriptElementKind.typeElement, [textPart(name)]) : undefined;
+        case StringLiteralCompletionKind.Cases:
+            return;

```

Comment:
I wasn't sure what kind of details should be returned here.

Reply:
I think you should do something like the above case for `StringLiteralCompletionKind.Types`, except I don't think the `kind` should be `ScriptElementKind.typeElement`, it should be `ScriptElementKind.string` instead. Not related to your PR: I don't know why the call above is using `ScriptElementKind.typeElement`, it looks weird to me that the `kind` in the `CompletionEntry` and `CompletionEntryDetails` return two different things for `StringLiteralCompletionKind.Types` case, do you know why that is @@andrewbranch.

Reply:
I came to the conclusion - that perhaps I don't need a completely separate type and I can just use `.Types`. At first, I wasn't sure what `.Types` is supposed to resolve so I was afraid of reusing it. WDYT?

Reply:
I don’t know the answer to @gabritto’s question, but I do think you can just use `.Types` for this.

Reply:
I think you can reuse it and fix the `CompletionEntryDetails.kind` weirdness. With any luck, if this turns out to be wrong for some reason, a test will break 😅